grails-app/views/contact.gsp

```diff
@@ -158,47 +139,16 @@
 </footer>
 </body>
 
-<script src="../assets/js/jquery-1.10.2.js" type="text/javascript"></script>
-<script src="../assets/js/jquery-ui-1.10.4.custom.min.js" type="text/javascript"></script>
-
-<script src="../bootstrap3/js/bootstrap.js" type="text/javascript"></script>
-
 <!--  Plugins -->
 <script src="../assets/js/ct-paper-checkbox.js"></script>
 <script src="../assets/js/ct-paper-radio.js"></script>
 <script src="../assets/js/bootstrap-select.js"></script>
 <script src="../assets/js/bootstrap-datepicker.js"></script>
 <script type="text/javascript" src="https://maps.googleapis.com/maps/api/js?v=3.exp"></script>
 
-<script src="../assets/js/ct-paper.js"></script>
-
-<script>
-  // Facebook Pixel Code Don't Delete
-  !function(f,b,e,v,n,t,s){if(f.fbq)return;n=f.fbq=function(){n.callMethod?
-    n.callMethod.apply(n,arguments):n.queue.push(arguments)};if(!f._fbq)f._fbq=n;
-    n.push=n;n.loaded=!0;n.version='2.0';n.queue=[];t=b.createElement(e);t.async=!0;
-    t.src=v;s=b.getElementsByTagName(e)[0];s.parentNode.insertBefore(t,s)}(window,
-    document,'script','//connect.facebook.net/en_US/fbevents.js');
-
-  try{
-    fbq('init', '111649226022273');
-    fbq('track', "PageView");
-
-  }catch(err) {
-    console.log('Facebook Track Error:', err);
-  }
-</script>
 <noscript>
```

Comment:
this isn't doing anything at the moment right. plus you removed the closing tag but left the rest of the <noscript> tag. best if you delete it completely